src/Microsoft.DotNet.Interactive.ExtensionLab/Microsoft.SqlTools.ServiceLayer/runtime.win-x86.runtime.native.Microsoft.SqlTools.ServiceLayer/Class1.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System;
```

Comment:
do we need this file?

Reply:
The temp file no. I will can it.  The class1.cs files, yes.